acceptance/setup/common/022_Remove_LD_PRELOAD.rb

```diff
@@ -0,0 +1,11 @@
+platforms = hosts.map{|val| val[:platform]}
```

Comment:
This new step needs to be listed in https://github.com/puppetlabs/pxp-agent/blob/master/acceptance/config/aio/options.rb#L13 or it won't be included by the ci:test:aio task